extensions/rest-client/deployment/src/test/java/io/quarkus/restclient/ft/RestClientAnnotationTest.java

```diff
@@ -0,0 +1,97 @@
+package io.quarkus.restclient.ft;
```

Comment:
Just a minor comment - It seems like this package is there for `fault tolerance` specific tests. Can we move this test on its own package?

Reply:
yes, sure. For example io.quarkus.restclient.rc would it be ok?

Reply:
I would put it in restclient, no need for rc

Reply:
ok

Reply:
+1 with @kenfinnigan

Reply:
Done!